sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -5,32 +5,38 @@
 - Adds abortSignal support throughout Sender and non-session Receivers.
   [PR 9233](https://github.com/Azure/azure-sdk-for-js/pull/9233)
   [PR 9284](https://github.com/Azure/azure-sdk-for-js/pull/9284)
-- Standardized methods on senders and receivers to use the `Messages` suffix and deal with multiple messages rather than have dedicated methods to deal with a single message.
-  [PR 9678](https://github.com/Azure/azure-sdk-for-js/pull/9678)
-- Standardized methods that peek and receive given number messages to use similar signature.
-  [PR 9798](https://github.com/Azure/azure-sdk-for-js/pull/9798)
 - Bug - Messages scheduled in parallel with the `scheduleMessage` method have the same sequence number in response.
   Fixed in [PR 9503](https://github.com/Azure/azure-sdk-for-js/pull/9503)
-- Management api updates (Includes breaking changes)
-
+- Management api updates
   - Following return types are changed to improve the API surface.
     - [Create,Get,Update]QueueResponse as QueueResponse, DeleteQueueResponse as Response, GetQueueRuntimeInfoResponse as QueueRuntimeInfoResponse.
       Similarly for topics, subscriptions and rules.
       [PR 9432](https://github.com/Azure/azure-sdk-for-js/pull/9432)
-    - Renamed `createdOn`, `accessedOn` and `modifiedOn` properties to `createdAt`, `accessedAt` and `modifiedAt`, updated the corresponding type from `ISO-8601 timestamp string` to the `Date` type in the responses for the `runtimeInfo` methods for Queue, Topic and Subscription.
-      [PR 9434](https://github.com/Azure/azure-sdk-for-js/pull/9434)
-      [PR 9807](https://github.com/Azure/azure-sdk-for-js/pull/9807)
-  - The property `top` in the options passed to any of the methods that get information for multiple entities like `getQueues` or `getQueuesRuntimeInfo` is renamed to `maxCount`.
-    [PR 9664](https://github.com/Azure/azure-sdk-for-js/pull/9664)
   - `OperationOptions` has been added for all the methods under `ServiceBusManagementClient`, this adds support for abortSignal, requestOptions when creating and sending HTTP requests.
     [PR 9654](https://github.com/Azure/azure-sdk-for-js/pull/9654)
+  - Fixed the bug where one cannot set `userProperties` in a correlation filter while using the `createRule()` method. [PR 9794](https://github.com/Azure/azure-sdk-for-js/pull/9794)
+
+### Breaking Changes
+
+- Standardized methods on senders and receivers to use the `Messages` suffix and deal with multiple messages rather than have dedicated methods to deal with a single message.
+  [PR 9678](https://github.com/Azure/azure-sdk-for-js/pull/9678)
+- Standardized methods that peek and receive given number messages to use similar signature.
+  [PR 9798](https://github.com/Azure/azure-sdk-for-js/pull/9798)
+- Removed `isReceivingMessages` method on the `Receiver`.
+  [PR 9875](https://github.com/Azure/azure-sdk-for-js/pull/9875)
```

Comment:
Enough? or requires an explanation?

Reply:
Requires explanation or at least a pointer to the issue that has the explanation

Reply:
```suggestion
- Removed `isReceivingMessages` method on the `Receiver` as per discussions in [Issue 9746](https://github.com/Azure/azure-sdk-for-js/issues/9746)
  [PR 9875](https://github.com/Azure/azure-sdk-for-js/pull/9875)
```

Reply:
Thanks!